spec/features/admin_requests_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'Viewing all requests' do
+  describe 'by a webauth user' do
+    before do
+      stub_current_user
+    end
+    it 'should list requests in tables' do
+      visit new_page_path(item_id: '1234', origin: 'GREEN', origin_location: 'STACKS')
+      click_button 'Send request'
+      visit new_page_path(item_id: '1235', origin: 'SAL-NEWARK', origin_location: 'STACKS')
+      click_button 'Send request'
```

Comment:
You can effectively do the same thing by calling
```ruby
Page.create!(item_id: '1234', origin: 'GREEN', origin_location: 'STACKS')
Page.create!(item_id: '1235', origin: 'SAL-NEWARK', origin_location: 'STACKS')
```
at the beginning of your test instead of needing the test to take 4 additional requests to complete.

On a side-note, it's probably about time we begin using `factory_girl` for creating the request objects so we don't have too much pain when the objects get more complex.  I'll submit an issue about doing that.